beamformer/test/prebeamform_reorder_test.py

```diff
@@ -95,9 +82,16 @@ def test_prebeamform_reorder_parametrised(batches, num_ants, num_channels, num_s
         The number of channels per stream is calculated from this value.
     num_samples_per_channel: int
         The number of time samples per frequency channel.
-    n_times_per_block: int
+    n_blocks: int
         Number of blocks to break the number of samples per channel into.
+
+    This test:
+        1. Populate a host-side array with random, numpy.uint16 data ranging from 0 to 16383-2.
+        2. Instantiate the prebeamformer_reorder_kernel and passes this input data to it.
+        3. Grab the output, reordered data.
+        4. Verify it relative to the input array.
```

Comment:
This is good information to go in the test's documentation, but some small suggestions regarding correctness.
```suggestion

    This test:
        1. Populate a host-side array with random data in the range of the relevant dtype.
        2. Instantiate the prebeamformer_reorder_kernel and pass this input data to it.
        3. Grab the output, reordered data.
        4. Verify it relative to the input array using a reference computed on the host.
```

The numerical range that you specified was correct for uint16, but not currently. It'll now assign [-128; 127) range numbers which int8 numbers can hold. The advantage to being a bit more vague and generic is that we don't need to update the docstrings (and avoid drifting out of sync) if we decide later to go to say 16-bit.

Reply:
Noted and fixed.